filters/filter_17_TrackParam/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 17,
+  "name": "AdGuard Tracking Parameters list",
+  "description": "Filter that enhances privacy by removing tracking parameters from URLs.",
+  "timeAdded": 1618479248382,
+  "homepage": "https://kb.adguard.com/general/adguard-ad-filters#removeparam-filter",
```

Comment:
We should change it to `https://kb.adguard.com/general/adguard-ad-filters` everywhere, there're no anchors on that page anyway.

Reply:
I've already asked @vbagirov to add anchors back (because homepage urls are used in [AdGuard Browser extension](https://uploads.adguard.com/slbdzdrx8u2o.png) for example). so should we really just simplify it?

Reply:
Well, the page is simplified for that, if you get to the anchor directly, you won't see the general description that's above